prez/config.py

```diff
@@ -83,8 +52,7 @@ class Settings(BaseSettings):
     top_level_classes: Optional[dict]
     collection_classes: Optional[dict]
     general_classes: Optional[dict]
-    enabled_prezs: Optional[list]
-    generate_support_graphs: bool = True
+    prez_flavours: Optional[list] = ["SpacePrez", "VocPrez", "CatPrez", "ProfilesPrez"]
```

Comment:
What is ProfilesPrez?

Reply:
Profiles/list of profiles are treated as objects the same as spatial, vocabulary, catalog data, so there's a Prez flavour for profiles which just displays profiles using the same mechanisms/code prez uses for the "standard" flavours

Reply:
Ok, I'm guessing it's reusing the logic of generic Prez to list https://gsq-prez.dev.kurrawong.ai/profiles?

Reply:
yes correct